core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2189,14 +2193,18 @@ public AdministrativeMonitor getAdministrativeMonitor(String id) {
      * @since 2.64
      */
     public List<AdministrativeMonitor> getActiveAdministrativeMonitors() {
-        return administrativeMonitors.stream().filter(m -> {
-            try {
-                return m.isEnabled() && m.isActivated();
-            } catch (Throwable x) {
-                LOGGER.log(Level.WARNING, null, x);
-                return false;
-            }
-        }).collect(Collectors.toList());
+        if (Jenkins.get().hasPermission(ADMINISTER)){
```

Comment:
but some `AdministrativeMonitors` are accessible with MANAGE (in this PR - see OldDataMonitor for example?)